build.sbt

```diff
@@ -35,7 +35,10 @@ libraryDependencies ++= Seq(
   "org.scalatestplus.play" %% "scalatestplus-play" % "5.1.0" % Test,
   "com.typesafe.play" %% "play-test" % "2.8.8" % "test",
   "com.typesafe.play" %% "play-json-joda" % "2.9.2",
-  "com.h2database" % "h2" % "1.4.200" % "test"
+  "com.h2database" % "h2" % "1.4.200" % "test",
+  "com.typesafe.slick" %% "slick"     % "3.3.2",
```

Comment:
isn't it already included in unicorn dependency?

Reply:
removed